lib/puppet/type/ec2_securitygroup.rb

```diff
@@ -49,6 +49,17 @@ def stringify_values(rules)
     end
   end
 
+  newproperty(:vpc_name) do
+    desc 'The name of the virtual private cloud (VPC) the security group belongs to, if applicable.'
+  end
+
+  newproperty(:id) do
```

Comment:
I would probably separate out the addition of the `:id` property for `ec2_securitygroup` to it's own commit as that's a standalone change.  Then adding `vpc_name` can be pared back to a smaller set of changes.